gum/backend-libdwarf/gumsymbolutil-libdwarf.c

```diff
@@ -344,6 +356,58 @@ gum_find_function (const gchar * name)
   return address;
 }
 
+static gboolean
+gum_find_nearest_symbol_by_address (const gpointer address,
+                                    GumNearestSymbolDetails * nearest)
+{
+  GHashTable * table;
+  gchar * name;
+  GList * keys;
+  GList * current;
```

Comment:
```suggestion
  GList * keys, * cur;
```

For consistency with other places where we walk such lists.